sample/src/main/java/com/alamkanak/weekview/sample/BaseActivity.java

```diff
@@ -211,4 +230,9 @@ public void onEmptyViewClicked(Calendar date) {
     public void onAddEventClicked(Calendar startTime, Calendar endTime) {
         Toast.makeText(this, "Add event clicked.", Toast.LENGTH_SHORT).show();
     }
+
+    @Override
+    public void onDrop(View view, Calendar date) {
+        Toast.makeText(this, "View dropped to " + getWeekView().getDateTimeInterpreter().interpretDate(date), Toast.LENGTH_SHORT).show();
```

Comment:
this info message appears in sample app, not in library, it was made in ordering to existing code styling